docs/notes/previous_versions/v1.8.4.rst

```diff
@@ -0,0 +1,15 @@
+Version 1.8.4
+^^^^^^^^^^^^^
+
+This release adds the following **feature**:
```

Comment:
```suggestion
This release adds the following **features**:
```

Reply:
It is only one feature. Should we use plural anyways?

Reply:
I thought about it. The suggestion was more for consistency, but singular also makes sense